templates/nextjs/src/app/layout.tsx

```diff
@@ -40,6 +50,10 @@ export default function RootLayout({ children }: RootLayoutProps) {
           <QueryClientProvider client={queryClient}>
             <FuelProvider
               fuelConfig={{
+                /**
+                 * The list of wallet connectors.
+                 * You can add or remove connectors from here based on your needs.
+                 */
```

Comment:
Maybe we should request some documentation for all supported connectors that we can link to here?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2772/commits/f6c90ff3c9bbdf7b2d033d7eed673c42ff4502ad